src/transaction_manager/transaction.cpp

```diff
@@ -62,7 +62,7 @@ Transaction::Transaction(dev::RLP const &_rlp, bool verify_strict) {
       break;
     }
   }
-  if (fields_processed != 9) {
+  if (fields_processed > 10) {
```

Comment:
From the above code, fields_processed only can be 9. Otherwise, that must include something go to else condition. What will cause that?

Reply:
Please check change in submodule https://github.com/Taraxa-project/taraxa-aleth/pull/22